HW_5/task 2.py

```diff
@@ -0,0 +1,12 @@
+a = int(input("Введите первое неотрицительное число "))
+b = int(input("Введите второе неотрицательно число "))
+
+
+def recursive_sum(a, b):
+    if a == 0:
+        return b
+    else:
+        return recursive_sum(a - 1, b + 1)
```

Comment:
Блок кода `else` не нужен, так как `return` завершает функцию.
```suggestion
    return recursive_sum(a - 1, b + 1)
```